src/components/Video/Video.tsx

```diff
@@ -50,7 +50,7 @@ export const Video = ({ url, title }: VideoProps) => {
             title={title}
             alt={title}
           />
-          <span>▶</span>
+          <span className="play-button">▶</span>
```

Comment:
should this `span` have an `aria-label`?

Reply:
the `span` should not have a `aria-label` but the `button` should have a `aria-label` or `aria=labelledby`

Reply:
thanks both, title on the img must have got this past the accessibility auditors. I agree Thomas. I've also removed the now redundant title.
I've also had to increase the size of the button, because I reduced it after testing to make it the same size as the youtube button, and making it smaller caused it to fail in contrast. increase size slightly.